Neos.Neos/Documentation/ExtendingNeos/CKEditor5WithNeos.md

```diff
@@ -0,0 +1,132 @@
+==========================
+Using CKEditor 5 with Neos
+==========================
+CKEditor 5 is delivered with neos-ui 1.3. Until neos-ui 2.0,
```

Comment:
This part needs to be updated with the current situation (CKE5 is the default since 2.0)